### ISSUE TITLE
TS Server fatal error:  Debug Failure. False expression.

### ISSUE THREAD

**Issue by thedomi** [microsoft/vscode#97142]
**TypeScript Version:** 3.8.3
**Steps to reproduce crash**
1. Happends every time I open VSCode. 

**TS Server Error Stack**
```
Error: Debug Failure. False expression.
    at ProjectService.applyChangesInOpenFiles (tsserver.js:143381:34)
    at Session.handlers.ts.createMapFromTemplate._a.<computed> (tsserver.js:144260:46)
    at tsserver.js:146003:88
    at IOSession.Session.executeWithRequestId (tsserver.js:145994:28)
    at IOSession.Session.executeCommand (tsserver.js:146003:33)
    at IOSession.Session.onMessage (tsserver.js:146027:35)
    at Interface.<anonymous> (tsserver.js:147342:27)
    at Interface.emit (events.js:203:13)
    at Interface._onLine (readline.js:316:10)
    at Interface._normalWrite (readline.js:461:12)
    at Socket.ondata (readline.js:172:10)
    at Socket.emit (events.js:203:13)
    at addChunk (_stream_readable.js:295:12)
    at readableAddChunk (_stream_readable.js:276:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at Pipe.onStreamRead (internal/stream_base_commons.js:166:17)
```

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)?

**Comment by mjbvz** [comment]
Also, duplicate of microsoft/TypeScript#38076

**Comment by thedomi** [comment]
Thanks @mjbvz . I've downloaded v 1.46 and the issue is still there. Do you need any other info on this issue (e.g., what's in https://github.com/microsoft/TypeScript/issues/38076#issuecomment-628166480)?

**Comment by mjbvz** [comment]
Yes please share the error you are seeing when using TypeScript 3.9.3+ (this is the version of TypeScript shipped in VS Code insiders)

**Comment by thedomi** [comment]
@mjbvz, I might have found the problem. 

Our project contains hundreds of .ts files (MPEG2 Transport Stream files, not TypeScript). I noticed that these were listed in the tslog file (I started sanitizing the tslog file as it contains code pieces from our project).

Adding the folder with the MPEG2 Transport Stream files to `tsconfig.json` -> `exclude` solved the problem. 

Thank you for your help with this.
